examples/example_plugin/example_plugin/templates/react/pages/models/add.jsx

```diff
@@ -1,7 +1,8 @@
+// TODO: what's the purpose of this file? Should it go away?
```

Comment:
Probably safe to remove, current implementation puts everything under `/ui/` not `/templates/react/`.